rotkehlchen/api/rest.py

```diff
@@ -3906,3 +3908,53 @@ def get_associated_locations(self) -> Response:
             result=_wrap_in_ok_result([str(location) for location in locations]),
             status_code=HTTPStatus.OK,
         )
+
+    def _query_kraken_staking_events(
+        self,
+        only_cache: bool,
+        filter_query: HistoryEventFilterQuery,
+    ) -> Dict[str, Any]:
+        exchanges_list = self.rotkehlchen.exchange_manager.connected_exchanges.get(
+            Location.KRAKEN,
+        )
+        message = ''
+        if exchanges_list is None:
+            events = []
+        else:
+            if only_cache is False:
+                for kraken_instance in exchanges_list:
+                    with_errors = kraken_instance.query_kraken_ledgers(   # type: ignore
+                        start_ts=filter_query.from_ts / KRAKEN_TS_MULTIPLIER,
```

Comment:
That's the issue that I commented before. Since the entries in the table queried have the higher resolution timestamps they filter have them adjusted to that resolution. Every other places that requires having a lower resolution needs to properly scale them.

If I change this I believe I would need to modify the prepare method of the filters

Reply:
Ah I see. Hmmm I think these conversions would make sense to be happening in the filter class eventually